### DISCUSSION TITLE
Are there any plans to open source JVM desktop?

### DISCUSSION THREAD

**Discussion by Nikoqqq** [GitLiveApp/firebase-kotlin-sdk#361] [General]
Hello. I am thinking about to develop Kotlin Multiplatform (android and desktop targets) app with firebase and i am not sure can i waiting for any Desktop implementation in this repository(in the coming months) or i should use Firebase REST to build it all by myself?

**Comment by nbransby** [comment]
We can possibly look into open sourcing our jvm port of firebase-android but would be looking for maintainers

**Comment by nbransby** [comment]
done: mhttps://github.com/GitLiveApp/firebase-java-sdk
